src/main/java/com/mk/cti/controller/PhoneController.java

```diff
@@ -35,14 +35,14 @@ public PhoneController(PhoneUtility phoneUtility, PhoneResponseService phoneResp
 
         PhoneResponse phoneResponse = phoneResponseService.findByPhoneNumber(phoneNumber);
 
-        if(phoneResponse == null)
-        {
+        if(phoneResponse == null) {
             phoneResponse = new PhoneResponse();
             phoneResponse.setCustomerId(0L);
             phoneResponse.setLoanId(0L);
             phoneResponse.setAuthCode("A093756186");
         }
-        if(phoneResponse == null) {
+        if(phoneResponse == null)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Change this condition so that it does not always evaluate to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS2583)